### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#500] [Esami & Esoneri M-Z]
### Traccia

<img width="562" alt="image" src="https://github.com/user-attachments/assets/c7c62a5d-aade-4a55-b523-34f25c806ef2">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/ac2c4edd-401e-4f21-95bf-4f17aa493acb)
